lua/goto-preview.lua

```diff
@@ -27,6 +27,7 @@ local M = {
     dismiss_on_move = false, -- Dismiss the floating window when moving the cursor.
     force_close = true, -- passed into vim.api.nvim_win_close's second argument. See :h nvim_win_close
     bufhidden = "wipe", -- the bufhidden option to set on the floating window. See :h bufhidden
+    nest_floating = true, -- Whether to nest floating windows
```

Comment:
I would maybe rename this to `stack_preview_windows` as that a bit more accurately describes what's actually happening.

Reply:
Cool. If we want to specify preview windows, should I add in a check for the `is-goto-preview-window` window variable?

Reply:
I've added the check for `is-goto-preview-window` and also made the option name `stack_floating_preview_windows` to be a bit more explicit; If someone moves their window to a non-floating split and then runs goto-preview again, a new floating preview window will open, even if it's still a preview-window.